modules/virt-about-upgrading-virt.adoc

```diff
@@ -23,6 +23,8 @@ connection. Most automatic updates complete within fifteen minutes.
 
 * Data volumes and their associated persistent volume claims are preserved during update.
 
+* The `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is now available to deploy as a standalone Operator from the *OperatorHub* in the {product-title} web console. If you are upgrading from {VirtProductName} 4.10.2 or later, you must move all nodes out of maintenance mode. If a node cannot be moved out of maintenance mode, install the standalone NMO Operator and replace the `nodemaintenances.nodemaintenance.kubevirt.io` custom resource (CR) with a `nodemaintenances.nodemaintenance.medik8s.io` CR. If you do not move nodes out of maintanance mode or update your NMO CR, your upgrade to {VirtProductName} {VirtVersion} will be blocked.
```

Comment:
I think this text should be formatted as a [NOTE] and placed below the assembly intro. Because it only applies to upgrading to 4.11, I feel it does not fit with the rest of the list items in the module "About updating OpenShift Virtualization"